docs/libcudacxx/extended_api/math/ceil_div.rst

```diff
@@ -20,10 +20,14 @@ The function computes the ceiling division between two integral or enumerator va
 
 Divides ``value`` by ``divisor``. If ``value`` is not a multiple of ``divisor`` rounds the result up to the next integer value.
 
+**Mandates**
+
+- ``T`` and ``U`` are integral types or enumerators.
```

Comment:
IIRC @miscco proposed the term "integer types" instead for those including `int128`
```suggestion
- ``T`` and ``U`` are integer types or enumerators.
```